grid.v3/registry/registry.go

```diff
@@ -55,6 +55,8 @@ type Registry struct {
 	keepAliveStats *keepAliveStats
 }
 
+var DefaultLeaseDuration = 60 * time.Second
```

Comment:
Can we remove this? The reason there is no global one is because it's per registry, and already set, so the user of the registry needs to do nothing to start using it. If the default is no good, the user of the registry can just change it right on the registry before making calls.

Though there should be documentation on the registry actually saying that it is set to a default of 60 seconds. I should of added that documentation, sorry.